Source/WebInspectorUI/UserInterface/Views/TreeOutline.js

```diff
@@ -234,25 +232,21 @@ WI.TreeOutline = class TreeOutline extends WI.Object
         this.children.push(child);
         this.hasChildren = true;
         child.parent = this;
-        child.treeOutline = this.treeOutline;
-        child.treeOutline._rememberTreeElement(child);
-
-        var current = child.children[0];
-        while (current) {
-            current.treeOutline = this.treeOutline;
-            current.treeOutline._rememberTreeElement(current);
-            current = current.traverseNextTreeElement(false, child, true);
-        }
+        this.treeOutline._rememberTreeElement(child);
```

Comment:
I feel like `_rememberTreeElement` should mention that it also remembers descendants. This was obvious before, because it used `traverseNextTreeElement` in a while loop.